test/insa.js

```diff
@@ -189,22 +189,47 @@ test('Kloster Unser Lieben Frauen to Magdeburg Hbf', co(function*(t) {
 	t.end()
 }))
 
-test('Magdeburg-Buckau to Magdeburg-Neustadt with stopover at Magdeburg Hbf', co(function*(t) {
-	const magdeburgBuckau = '8013456'
-	const magdeburgNeustadt = '8010226'
-	const magdeburgHbf = '8010224'
-	const [journey] = yield client.journeys(magdeburgBuckau, magdeburgNeustadt, {
-		via: magdeburgHbf,
+test('journeys: via works – with detour', co(function* (t) {
+	// Going from Magdeburg, Hasselbachplatz (Sternstr.) (Tram/Bus) to Stendal via Dessau without detour
+	// is currently impossible. We check if the routing engine computes a detour.
+	const hasselbachplatzSternstrasse = '000006545'
+	const stendal = '008010334'
+	const dessau = '008010077'
```

Comment:
I guess it is a matter of choice if we want to strip the leading `0`s. The API doesn't return these consistently anyways.

Reply:
I'd just leave them like they are right now. I'll have to tackle this anyways when rewriting all the tests (which is my intention), so IMHO there's no point in changing this in this PR.

Reply:
If you *rewrite* the tests, make sure to do it in tiny steps so we don't lose details like individual assumptions or comments.